presto-main/src/main/java/io/prestosql/sql/rewrite/ExplainRewrite.java

```diff
@@ -74,7 +75,7 @@ public Visitor(
                 WarningCollector warningCollector)
         {
             this.session = requireNonNull(session, "session is null");
-            this.queryPreparer = new QueryPreparer(requireNonNull(parser, "queryPreparer is null"));
+            this.queryPreparer = new QueryPreparer(requireNonNull(parser, "queryPreparer is null"), new NonModifyingStatementAccessControl());
```

Comment:
Would it be possible to pass `StatementAccessControlManager` here?
Otherwise explain won't show actual query that get's executed.

Reply:
Shall check this

Reply:
Shall check and update